src/data/astarte.rs

```diff
@@ -32,7 +32,7 @@ pub struct Astarte {
 
 #[async_trait]
 impl Publisher for Astarte {
-    async fn send_object<T>(
+    async fn send_object<T: 'static>(
```

Comment:
If I understand correctly, isn't this saying that _all_ payloads that we pass to `send_object` have a `static` lifetime (so they're valid throughout the course of all the program)? Is this actually the case?

Reply:
@rbino It related to this case [trait-bound](https://doc.rust-lang.org/rust-by-example/scope/lifetime/static_lifetime.html#trait-bound)